project-1/main.py

```diff
@@ -0,0 +1,57 @@
+import gspread
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+
+driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
+link = "https://www.food.com/ideas/top-comfort-food-recipes-6929#c-791290"
+driver.get(url=link)
+
+gc = gspread.service_account(filename='my-project-1202.json')
+
+sh = gc.open("project 1202")
+
+worksheet = sh.sheet1
+
+print(worksheet.acell("A2").value)
+
+# Formatting first column
```

Comment:
If this is your final version, please remove the redundant comments from your code